x/ecocredit/client/testsuite/query.go

```diff
@@ -89,7 +89,7 @@ func (s *IntegrationTestSuite) TestQueryClassInfo() {
 		expectedClassInfo *ecocredit.ClassInfo
 	}{
 		{
-			name:           "missing credit class",
```

Comment:
technically not missing specifically the credit class - when cmd's have `exactArgs` it will fail immediately if len(argsPassed) isn't equal to amount of exactArgs.